backend/src/personnel/ro/bcws/bcws-personnel-roles.ro.ts

```diff
@@ -31,4 +31,12 @@ export class BcwsPersonnelRoleRO {
     enum: ExperienceLevel,
   })
   expLevel: ExperienceLevel;
+
+  @ApiProperty({
+    description: 'Rank of choice for role (first, second, etc.)',
+    required: true,
+    example: 1,
+    type: 'number'
+  })
+  rank: number;
```

Comment:
I think this will need to be optional since we'll have null for some of the roles' ranks